carma_wm_ctrl/src/WMBroadcaster.cpp

```diff
@@ -868,18 +890,35 @@ lanelet::BasicPoint2d curr_pos;
         for(auto id : active_geofence_llt_ids_) 
         {
           if (id == current_llt.id())
-          {
-            outgoing_geof.type = 1;
+          {           
             outgoing_geof.is_on_active_geofence = true;
             for (auto regem: current_llt.regulatoryElements())
               {
                   if (regem->attribute(lanelet::AttributeName::Subtype).value().compare(lanelet::DigitalSpeedLimit::RuleName) == 0)
                   {
                     lanelet::DigitalSpeedLimitPtr speed =  std::dynamic_pointer_cast<lanelet::DigitalSpeedLimit>
                     (current_map_->regulatoryElementLayer.get(regem->id()));
-                   outgoing_geof.value = speed->speed_limit_.value();
+                    outgoing_geof.value = speed->speed_limit_.value();
+                    outgoing_geof.advisory_speed = speed->speed_limit_.value(); 
```

Comment:
What is the difference in this outgoing message between .value and .advisory_speed?

Reply:
value is only used for Speed Limit event type. The Advisory speed is for the closed event type. The UI is taking data from different speed fields depends on the event types.

Reply:
ok sounds good

Reply:
https://github.com/usdot-fhwa-stol/carma-platform/pull/1197/commits/761f745a65683501b99f5387e17f2388de375e83